runner/src/run.rs

```diff
@@ -75,6 +75,20 @@ fn get_config(args: &RunArgs) -> Config {
     cfg
 }
 
+fn is_qemu_version_6() -> bool {
+
+    let output = Command::new("qemu-system-riscv64")
+        .arg("--version")
+        .output()
+        .expect("Failed to execute command");
```

Comment:
Be a bit more specific here, it might not be clear for the user what command failed.

Something like "Failed to execute `qemu-system-riscv64 --version`" would be more helpful to diagnose an issue.